contentcuration/contentcuration/frontend/channelList/views/ChannelSet/ChannelSetModal.vue

```diff
@@ -245,7 +248,9 @@
       },
     },
     beforeMount() {
-      return this.verifyChannelSet(this.channelSetId);
+      if (this.channelSetId) {
+        return this.verifyChannelSet(this.channelSetId);
+      } else this.setup();
```

Comment:
Can we keep the syntax consistent here?
```js
if (this.channelSetId) {
    return this.verifyChannelSet(this.channelSetId);
} else {
    this.setup();
}
```
Also, is there any reason we are returning `this.verifyChannelSet`?

Reply:
What would be the significance in returning its value to the `beforeMount()` hook?